### ISSUE TITLE
Update Profile Bug.  Display name not displaying? Can't update through edit tool.

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#325]
If we don't think this affects users (as opposed to an admin editing someone's profile) make this medium priority.

I was on http://gbeadmin.webfactional.com/profile/manage and noticed that Lili vonSchtupp (first line) doesn't have a display name, so I clicked "Update" and ended up at http://gbeadmin.webfactional.com/profile/admin/563

where she does, clearly have a display name... but not some other information.

I added a phone number, I turned off "Display Hotel info", I added some interests, and I switched 'Staying at the hotel?' to "yes".

I hit "Update my account" and got barf...

ValueError at /profile/admin/563
The view gbe.views.admin_profile didn't return an HttpResponse object.
Request Method:	POST
Request URL:	http://gbeadmin.webfactional.com/profile/admin/563
Django Version:	1.6.5
Exception Type:	ValueError
Exception Value:	
The view gbe.views.admin_profile didn't return an HttpResponse object.
Exception Location:	/home/gbeadmin/webapps/gbetest/lib/python2.7/django/core/handlers/base.py in get_response, line 130
Python Executable:	/usr/local/bin/python
Python Version:	2.7.9
Python Path:	
['/home/gbeadmin/webapps/gbetest',
 '/home/gbeadmin/webapps/gbetest/expo',
 '/home/gbeadmin/lib/python2.7/psutil-2.1.1-py2.7-linux-x86_64.egg',
 '/home/gbeadmin/lib/python2.7/pip-1.5.6-py2.7.egg',
 '/home/gbeadmin',
 '/home/gbeadmin/lib/python2.7/pytz-2014.4-py2.7.egg',
 '/home/gbeadmin/lib/python2.7',
 '/home/gbeadmin/webapps/gbetest/lib/python2.7',
 '/usr/local/lib/python27.zip',
 '/usr/local/lib/python2.7',
 '/usr/local/lib/python2.7/plat-linux2',
 '/usr/local/lib/python2.7/lib-tk',
 '/usr/local/lib/python2.7/lib-old',
 '/usr/local/lib/python2.7/lib-dynload',
 '/home/gbeadmin/.local/lib/python2.7/site-packages',
 '/usr/local/lib/python2.7/site-packages',
 '/usr/local/lib/python2.7/site-packages/PIL']
Server time:	Fri, 6 Feb 2015 14:03:01 +0000

**Comment by burlexpo** [comment]
OK, and I can update my own profile through my own homepage, so the bug is in the Update admin tool somewhere.

**Comment by bethlakshmi** [comment]
OK... so a few notes prior to close:
 1) if users circumvent the workflow for first login, the weirdness of most of Lili's account will happen - it's technically possible to create a login and then NOT give us profile info, in which case, display name will be blank.  We have a sneaky trick in the profile update page - if you don't have a display name, we autogenerate one for you and suggest it.

 2) there was a very legit but which I just fixed on the live site - when there was an error in the form (for example, if the BPT ticket info or phone number was missing) - then it just bombed as described above.  Fixed that -- now it gives you a nice error telling you what you did wrong.
